src/precompile/bn254.rs

```diff
@@ -1,22 +1,24 @@
 use revm::precompile::{
-    bn128::{self, run_pair, PAIR_ELEMENT_LEN},
-    PrecompileError, PrecompileResult, PrecompileWithAddress,
+    bn254::{self, run_pair, PAIR_ELEMENT_LEN},
+    PrecompileError, PrecompileResult,
 };
 
 pub mod pair {
     use super::*;
 
-    pub use bn128::pair::{ADDRESS, ISTANBUL_PAIR_BASE, ISTANBUL_PAIR_PER_POINT};
+    pub use bn254::pair::{ADDRESS, ISTANBUL_PAIR_BASE, ISTANBUL_PAIR_PER_POINT};
```

Comment:
Just to confim `bn128` and `bn254` is the same curve and the operations are unchanged? This is just a nme change?

Reply:
yes this was updated in revm, mainly quoting this [comment](https://github.com/ethereum/EIPs/pull/10029#issue-3240867404) on the EIP.